corral/cmd/cmd_info.pony

```diff
@@ -5,11 +5,7 @@ use "../util"
 
 primitive CmdInfo
   fun apply(ctx: Context, cmd: Command) =>
-    //ctx.log.info("info: " + cmd.string())
```

Comment:
Please leave the `log.info` calls in for debugging. (Even the commented out ones) I wanted to work on making that a separately controllable output stream, but that's still WIP.

Reply:
Sorry, my bad.